integration-tests/jest.test.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
   testRegex: '(.*)\\.it\\.js$',
+  setupTestFrameworkScriptFile: 'jest-chain',
```

Comment:
Introducing `jest-chain` dependency for multiple assertions on the same variable.

Reply:
do we really need this? 
`jest-chain` should have come right out of the box

Reply:
Ahh, interesting. I checked the docs real quick and guess this is what you refer to, @adnasa ?

https://jestjs.io/docs/en/api#testeachtable-name-fn-timeout

If so I definitely agree on using the core part instead of adding an unnecessary dependency

Reply:
yes it was. sorry for the late reply